debian/clickhouse-server.postinst

```diff
@@ -101,9 +101,7 @@ Please fix this and reinstall this package." >&2
 
     if [ ! -d ${CLICKHOUSE_LOGDIR} ]; then
         mkdir -p ${CLICKHOUSE_LOGDIR}
-        chown root:${CLICKHOUSE_GROUP} ${CLICKHOUSE_LOGDIR}
-        # Allow everyone to read logs, root and clickhouse to read-write
-        chmod 775 ${CLICKHOUSE_LOGDIR}
+        chown ${CLICKHOUSE_USER}:${CLICKHOUSE_GROUP} ${CLICKHOUSE_LOGDIR}
```

Comment:
But this modification is still questionable.

Reply:
I don't see the difference of having 775 with root:clickhouse and 755 with clickhouse:clickhouse

Reply:
It is important for somewhat reason.
It's unlikely that developer implemented it that way without some motivation (but they forgot to add comments).

Reply:
ok.. I'll rollback it. Luckily there is still one line left to commit ;)